charts/camunda-platform-8.8/templates/orchestration/migration-data-configmap.yaml

```
@@ -97,7 +97,10 @@ data:
             retry-delay-multiplier: {{ .Values.orchestration.migration.data.tumetrics.retry.retryDelayMultiplier }}
         tasks:
           batch-size: {{ .Values.orchestration.migration.data.tasks.batchSize }}
-          importer-finished-timeout: {{ .Values.orchestration.migration.data.process.importerFinishedTimeout }}
+          importer-finished-timeout: {{ .Values.orchestration.migration.data.tasks.importerFinishedTimeout }}
+          {{- if .Values.orchestration.history.retention.enabled }}
+          legacy-index-retention-age: {{ .Values.orchestration.migration.data.tasks.legacyIndexRetentionAge | default .Values.orchestration.history.retention.minimumAge | quote }}
```

Thanks for the PR!

There is no need to add a default on top of another default. Here is what I would do:
```suggestion
          legacy-index-retention-age: {{ .Values.orchestration.migration.data.tasks.legacyIndexRetentionAge | quote }}
```

@panagiotisgts do both values have to be the same for the orchestration cluster?

---

@hamza-m-masood No, what I actually want is that the `legacy-index-retention-age` should be the same as `.Values.orchestration.history.retention.minimumAge` by default or be overriden by the `.Values.orchestration.migration.data.tasks.legacyIndexRetentionAge` isn't that what it accomplishes?

---

Please mention this in the comment of this value